qnq-frontend/package-lock.json

```diff
@@ -6655,6 +6664,23 @@
       "resolved": "https://registry.npmjs.org/react-native-iphone-x-helper/-/react-native-iphone-x-helper-1.3.1.tgz",
       "integrity": "sha512-HOf0jzRnq2/aFUcdCJ9w9JGzN3gdEg0zFE4FyYlp4jtidqU03D5X7ZegGKfT1EWteR0gPBGp9ye5T5FvSWi9Yg=="
     },
+<<<<<<< HEAD
+    "react-native-paper": {
+      "version": "4.7.0",
+      "resolved": "https://registry.npmjs.org/react-native-paper/-/react-native-paper-4.7.0.tgz",
+      "integrity": "sha512-RZwYZFFKry8LHHBYf5NENVaU2HaApERNcBLVzt4IE9AS0I+NkkLUyjKQYuLrgTF2EaZVEdeyUJwO7uQH/iNrnQ==",
+      "requires": {
+        "@callstack/react-theme-provider": "^3.0.5",
+        "color": "^3.1.2",
+        "react-native-iphone-x-helper": "^1.3.1"
+      }
+=======
+    "react-native-maps": {
+      "version": "0.27.1",
+      "resolved": "https://registry.npmjs.org/react-native-maps/-/react-native-maps-0.27.1.tgz",
+      "integrity": "sha512-HygBkZBecTnIVRYrSiLRAvu4OmXOYso/A7c6Cy73HkOh9CgGV8Ap5eBea24tvmFGptjj5Hg8AJ94/YbmWK1Okw=="
+>>>>>>> e36c60143fea3fb72abf0eeed59049f2c702e419
+    },
```

Comment:
Remove these conflict markers by running `npm install` in qnq-frontend folder again.